### DISCUSSION TITLE
Enhancement: Discord Rich Presence support

### DISCUSSION THREAD

**Discussion by superguyoffa** [FunkinCrew/Funkin#1606] [Ideas]
For those unaware, Discord Rich Presence is used by some games to show your friends more information about your game on Discord. This could be used in Friday Night Funkin' to show you what track, week or even mod you are currently playing.

This is more of a thing that's nice to have rather than a necessity.

**Comment by notkaif** [comment]
Heck yeah

**Comment by TentaRJ** [comment]
Maybe...

**Comment by GStudiosX2** [comment]
Sounds cool but kinda useless and I mean you can add it yourself if you want
